### PR TITLE
comeon module

### PR DESCRIPTION
**Pull request by eklundkristoffer**

### CODE REVIEW

File: build/modules/casino-modules/COMEON/services/registrationService.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    phoneNumber: user.phone.split('-')[1],
+    year: user.birth.year,
+    month: user.birth.month,
+    day: user.birth.day,
+    gender: user.gender,
+    country: user.countryCode,
+    currency: user.currency,
+    state: "",
+    voucherCode: "",
+    deviceId: 1468621770,
```
Comment (henke164) [inline build/modules/casino-modules/COMEON/services/registrationService.js]: Should this be hard coded ?

File: build/modules/casino-modules/COMEON/services/slotSessionProvider.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// THIS IS EXAMPLE CODE
+const fetch = require("node-fetch");
+const { userAgentHeader, jsonHeader } = require("../../../utilities/headers");
+
+async function getSlotSession(slotId, casinoCookies) {
```
Comment (henke164) [inline build/modules/casino-modules/COMEON/services/slotSessionProvider.js:4]: not finished yet

File: build/modules/casino-modules/COMEON/services/loginService.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+// THIS IS EXAMPLE CODE
+const fetch = require("node-fetch");
+const {
+  getCookieStringFromResponse
+} = require("../../../utilities/cookieHelper");
+const { userAgentHeader, jsonHeader } = require("../../../utilities/headers");
+
+async function login(user) {
+  const body = {
```
Comment (henke164) [inline build/modules/casino-modules/COMEON/services/loginService.js:10]: are captcha & deviceid not required?

Comment (eklundkristoffer) [reply build/modules/casino-modules/COMEON/services/loginService.js:10]: deviceId can be blank upon login but not on registration, captchaToken can be blank as well it seems.

**Review by henke164 — APPROVED** [review]

**Pull request merged**